syncstorage-spanner/src/batch.rs

```diff
@@ -98,7 +98,7 @@ pub async fn append_async(db: &SpannerDb, params: params::AppendToBatch) -> Resu
     if !exists {
         // NOTE: db tests expects this but it doesn't seem necessary w/ the
         // handler validating the batch before appends
-        Err(DbErrorKind::BatchNotFound)?
+        return Err(DbError::batch_not_found());
```

Comment:
I know that adding the `?` does the same thing, but I agree that this is more readable, particularly when fast reading the code.